python/pipeline/experiment.py

```diff
@@ -620,11 +639,11 @@ class ProjectorConfig(dj.Lookup):
     refresh_rate                : float                     # refresh rate in Hz
 
     """
-    contents = [
-        [0, 4, 2, 3, 60],
-        [1, 4, 4, 2, 60],
-        [2, 4, 4, 2, 30]
-    ]
+    # contents = [
```

Comment:
much rather you just remove this

Reply:
that makes sense.  we were getting the following error, so we commented it out until further discussion.  any suggestions?

```
---------------------------------------------------------------------------
IntegrityError                            Traceback (most recent call last)
<ipython-input-2-48638f745193> in <module>
----> 1 from pipeline import pupil

/data/pipeline/python/pipeline/pupil.py in <module>
     37 from .utils.eye_tracking import PupilTracker, ManualTracker
     38 from . import config
---> 39 from . import experiment, notify, shared
     40 from .exceptions import PipelineException
     41 

/data/pipeline/python/pipeline/experiment.py in <module>
    629 
    630 @schema
--> 631 class ProjectorConfig(dj.Lookup):
    632     definition = """
    633     # projector configuration

/usr/local/lib/python3.6/dist-packages/datajoint/schemas.py in __call__(self, cls, context)
    215         if issubclass(cls, Part):
    216             raise DataJointError('The schema decorator should not be applied to Part relations')
--> 217         self.process_table_class(cls, context=dict(context, self=cls, **{cls.__name__: cls}))
    218 
    219         # Process part relations

/usr/local/lib/python3.6/dist-packages/datajoint/schemas.py in process_table_class(self, table_class, context, assert_declared)
    204                             table=instance.__class__.__name__))
    205                 else:
--> 206                     instance.insert(contents, skip_duplicates=True)
    207 
    208     def __call__(self, cls, *, context=None):

/usr/local/lib/python3.6/dist-packages/datajoint/table.py in insert(self, rows, replace, skip_duplicates, ignore_extra_fields, allow_direct_insert)
    345                                if skip_duplicates else ''))
    346                 self.connection.query(query, args=list(
--> 347                     itertools.chain.from_iterable((v for v in r['values'] if v is not None) for r in rows)))
    348             except UnknownAttributeError as err:
    349                 raise err.suggest('To ignore extra fields in insert, set ignore_extra_fields=True') from None

/usr/local/lib/python3.6/dist-packages/datajoint/connection.py in query(self, query, args, as_dict, suppress_warnings, reconnect)
    212         cursor = self._conn.cursor(cursor=cursor_class)
    213         try:
--> 214             self._execute_query(cursor, query, args, cursor_class, suppress_warnings)
    215         except errors.LostConnectionError:
    216             if not reconnect:

/usr/local/lib/python3.6/dist-packages/datajoint/connection.py in _execute_query(cursor, query, args, cursor_class, suppress_warnings)
    194                 cursor.execute(query, args)
    195         except client.err.Error as err:
--> 196             raise translate_query_error(err, query) from None
    197 
    198     def query(self, query, args=(), *, as_dict=False, suppress_warnings=True, reconnect=None):

IntegrityError: Cannot add or update a child row: a foreign key constraint fails (`pipeline_experiment`.`#projector_config`, CONSTRAINT `#projector_config_ibfk_1` FOREIGN KEY (`channel_1`) REFERENCES `#projector_color` (`color`) ON UPDATE CASCADE)
```

Reply:
I think it's ok to remove, it will look very different.